app/vpc.tf

```diff
@@ -4,6 +4,7 @@ locals {
 
 module "vpc" {
   source = "terraform-aws-modules/vpc/aws"
+  version = ">= 2.4.0, < 3.0.0"
```

Comment:
I'm pretty sure we need to enforce the second half (`< 3.0.0`) as well, so that nothing newer gets used (since we're operating withing a finite window of compatible versions). I could certainly be wrong.

Reply:
https://www.terraform.io/language/expressions/version-constraints#-3

Reply:
I've just pushed an update with the change to `~2.4`.